### PR TITLE
Closure Compilation support

### PR DESCRIPTION
**Pull request by azakus**

### CODE REVIEW

File: lib/mixins/property-accessors.html [hunk 1]
```diff
@@ -563,6 +568,8 @@
 
     }
 
+    return PropertyAccessors;
```
Comment (kevinpschaaf) [inline lib/mixins/property-accessors.html]: Investigate if separating return from class definition is necessary; not used above.

File: lib/utils/templatize.html [hunk 2]
```diff
@@ -40,7 +40,12 @@
     }
 
     // Base class for TemplateInstance's
-    class TemplateInstanceBase extends Polymer.PropertyEffects(class{}) {
+    /**
+     * @constructor
+     * @mixes {Polymer_PropertyEffects}
```
Comment (kevinpschaaf) [inline lib/utils/templatize.html]: `@implements`

File: lib/mixins/property-accessors.html (hunk 1 above)
Comment (azakus) [inline lib/mixins/property-accessors.html]: This is actually necessary :(

**Comment by kevinpschaaf** [comment]
LGTM

**Pull request merged**